mods/books/init.lua

```diff
@@ -0,0 +1,89 @@
+books = {}
+
+dofile(minetest.get_modpath("books") .. "/craftitems.lua")
+dofile(minetest.get_modpath("books") .. "/nodes.lua")
+dofile(minetest.get_modpath("books") .. "/crafting.lua")
+
+if mg_name == "v6" then
```

Comment:
You need to 'get mapgen setting' first https://github.com/minetest/minetest_game/blob/master/mods/default/mapgen.lua#L1471